src/design/icons/icons.stories.mdx

```diff
@@ -2,7 +2,11 @@ import { Meta, IconGallery, IconItem } from '@storybook/addon-docs/blocks';
 import { extname, relative } from 'path';
 // Import whole directory of icons together
 // @see https://webpack.js.org/guides/dependency-management/#context-module-api
-const iconDir = require.context('../../assets/icons', false, /\.svg$/);
+const iconDir = require.context(
+  '!!@svgr/webpack!../../assets/icons',
```

Comment:
Could you add a comment explaining this svgr syntax, or linking to some docs?

Reply:
Sure, this is more webpack syntax than svgr but I've linked to docs for both now.